README.md

```diff
@@ -182,29 +182,165 @@ const agent: Shifter =  shifter.setIp();
 const agent: Shifter = shifter.setIp({ countries: [EShifterCountry.FRANCE] });
 ```
 
-#### Strategy: Change ip every requests
+#### Strategy: Change IP every requests
 
 ```typescript
 import { Shifter, EStrategyMode, EShifterCountry } from "@tictactrip/luminator";
 
 const shifter: Shifter = new Shifter({
-  proxy: {
-    host: '76.34.12.53',
-    port: 17664,
-  },
-  strategy: {
-    mode: EStrategyMode.CHANGE_IP_EVERY_REQUESTS,
-    mapping: {
-        fr: [17643, 17644],
-        es: [17645, 17646],
+    proxy: {
+        host: '76.34.12.53',
+        port: 17664,
+    },
+    strategy: {
+        mode: EStrategyMode.CHANGE_IP_EVERY_REQUESTS,
+        mapping: {
+            fr: [17643, 17644],
+            es: [17645, 17646],
+        },
+    },
+});
+
+const requestConfig = {
+    method: 'get',
+    baseURL: 'https://lumtest.com',
+    url: '/myip.json',
+}
+
+const response1 = await shifter.fetch(requestConfig);
+const response2 = await shifter.fetch(requestConfig);
+
+console.log(response1.data);
+console.log(response2.data);
```

Comment:
`console.log()` is only useful if a sample output is provided, and in this case, the interesting part would be the IP used for query, which I'm unsure is present in the response (maybe?).